Install wireguard-tools from feeds. Fix pythonpath bug.

amstewart opened the pull request:
### Summary of Changes

* Change the `_wireguard_config` module to install `wireguard-tools` from a native NILRT IPK, rather than from the debian package repos.
* Fix a bug in the `nilrt-snac` wrapper script that caused local paths to pollute module execution and mess with other search paths.


### Justification

[AB#2769306](https://dev.azure.com/ni/94b22d7b-ad7b-4f5e-88f0-867910f91c94/_workitems/edit/2769306)

The path fix is incidental.


### Testing

* [x] Installed these changes to to a local NILRT VM (scarthgap) and tested that the wireguard-tools IPK is now coming from the feeds, and that the script works even when run from a local source checkout of nilrt-snac.


### Procedure

* [x] I certify that the contents of this pull request complies with the [Developer Certificate of Origin](https://github.com/ni/nilrt-snac/blob/master/docs/CONTRIBUTING.md).

tkrebes reviewed (approved):
Changes look good to me.

amstewart commented on src/nilrt-snac:
```diff
@@ -5,4 +5,5 @@ export SCRIPTPATH=$(dirname $0)
 popd > /dev/null
-PYTHONPATH=${PYTHONPATH}:${PYTHONSCRIPTPATH} python3 -m nilrt_snac $@
+
+PYTHONPATH=${PYTHONSCRIPTPATH}:${PYTHONPATH} python3 -P -m nilrt_snac $@
```
I created issue #37 to track this, but won't fix as part of this PR.

AlexHearnNI commented on nilrt_snac/_configs/_wireguard_config.py:
```diff
@@ -25,13 +23,8 @@ def configure(self, args: argparse.Namespace) -> None:
-            opkg_conf.save(dry_run)
-        self._opkg_helper.install("./wireguard-tools.deb", force_reinstall=True)
+
+        self._opkg_helper.install("wireguard-tools", force_reinstall=True)
```
Why do we need `force_reinstall`? Was it because we were using the debian package?

amstewart replied:
Not sure. I'll try to remove that param.

amstewart commented:
# Patch V2
* Removed the `force_reinstall` param. The defaults seem to work fine, even when the configure workflow is re-run. @AlexHearnNI 
* Added a patch to fix #20 
* Added a patch to fix #37 @tkrebes
